src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/images/ImageScalerTest.java

```diff
@@ -19,29 +19,121 @@
 @RunWith (CdiTestRunner.class)
 public class ImageScalerTest {
   
-  
   @Inject
   private ImageReader imageReader;
-
+  
   @Inject
   private ImageScaler imageScaler;
 
   @Test
-  public void testScaleMaxSize() throws IOException {
+  public void testScaleToFitLandscape() throws IOException {
     assertNotNull(imageScaler);
-    try (InputStream imageStream = getClass().getClassLoader().getResourceAsStream("test-image-1000.jpg")) {
+    try (InputStream imageStream = getClass().getClassLoader().getResourceAsStream("test-image-1000-667.jpg")) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "test-image-1000-667.jpg" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)